<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshImport</class>
 <widget class="QDialog" name="MeshImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeshImport</string>
  </property>
  <widget class="QCheckBox" name="checkBox_convert_axis">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert Axis</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_convert_scale">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>58</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert Scale</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_split_points">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Split Points</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_generate_tangents">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Tangents</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_triangulate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Triangulate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_convert_textures">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert Textures</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>checkBox_convert_axis</tabstop>
  <tabstop>checkBox_convert_scale</tabstop>
  <tabstop>checkBox_split_points</tabstop>
  <tabstop>checkBox_generate_tangents</tabstop>
  <tabstop>checkBox_triangulate</tabstop>
  <tabstop>checkBox_convert_textures</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
